server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/eml/sameInlinedImages.eml

```diff
@@ -0,0 +1,294 @@
+Return-Path: <qnguyen@linagora.com>
```

Comment:
Anonimation : 
 - Try avoid putting your real mail (spam...)
 - Don't internals of Linagora mail routing in here